### PR TITLE
Turn Jake's unfinished post to draft

### PR DESCRIPTION
**Pull request by gewenyu99**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

**Comment by gewenyu99** [comment]
https://github.com/appwrite/website/assets/29069505/ea551102-c761-414f-bf5b-c52d3a0180e8

Now it's a draft, hidden from nav

File: src/routes/blog/+page.svelte [hunk 1]
```diff
@@ -180,7 +180,8 @@
                                 {@const author = data.authors.find(
                                     (author) => author.slug === post.author
                                 )}
-                                {#if author}
+                                {@const draft = post.draft}
```
Comment (stnguyen90) [inline src/routes/blog/+page.svelte:183]: why do we need to declare this variable?

Comment (gewenyu99) [reply src/routes/blog/+page.svelte:183]: Do you think the code will look cleaner with or without?

Comment (stnguyen90) [reply src/routes/blog/+page.svelte:183]: I think it's fine without.

Comment (gewenyu99) [reply src/routes/blog/+page.svelte]: Updated

File: src/routes/blog/+page.ts [hunk 2]
```diff
@@ -27,7 +27,8 @@ export function load() {
                 timeToRead: frontmatter.timeToRead,
                 author: frontmatter.author,
                 category: frontmatter.category,
-                href: `${base}/blog/post/${postName}`
+                href: `${base}/blog/post/${postName}`,
+                draft:frontmatter.draft
```
Comment (stnguyen90) [inline src/routes/blog/+page.ts:31]:
```suggestion
                draft: frontmatter.draft
```

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

**Review by stnguyen90 — APPROVED** [review]

**Pull request merged**